src/payment/strategies/moneris/moneris-payment-strategy.ts

```diff
@@ -126,6 +126,10 @@ export default class MonerisPaymentStrategy implements PaymentStrategy {
             frameref?.postMessage('tokenize', this._monerisURL(!!testMode));
 
             this._windowEventListener = (response: MessageEvent) => {
+                if (typeof(response.data) === 'object') {
```

Comment:
it's safer to test that values are of the type that we want, rather than if they are one of the type we do not.

e.g.
`if (typeof(response.data) !== 'string')`

Reply:
♻️ @leeBigCommerce